server/sonar-server/src/main/java/org/sonar/server/user/ws/FavouritesWs.java

```diff
@@ -22,11 +22,13 @@
 import org.sonar.api.server.ws.RailsHandler;
 import org.sonar.api.server.ws.WebService;
 
-public class FavoritesWs implements WebService {
+import static org.sonar.api.server.ws.RailsHandler.addFormatParam;
+
+public class FavouritesWs implements WebService {
 
   @Override
   public void define(Context context) {
-    NewController controller = context.createController("api/favorites");
+    NewController controller = context.createController("api/favourites");
```

Comment:
checks `routes.rb`. I think that we want to handle backward compatibility and the 2 should work